views/guide.lavender

```diff
@@ -89,8 +89,9 @@ block content
           h2#github via Github      
           p
             | Add the lavender submodule with 
-            code git submodule add git@github.com:golavender/lavender.git (folder to 
-            | clone to) and include it with 
+            code git submodule add git@github.com:golavender/lavender.git (folder to
+            code clone to)
```

Comment:
Thanks Cara! I'll definitely do that in the future. I'm gonna go through
the whole, longer process for this one again this time, (even though it's
just a teensy change) just for the sake of my own learning. I want to make
sure I can commit changes and do pull requests correctly...



On Mon, Mar 9, 2015 at 12:48 PM, Cara Simoneau <notifications@github.com>
wrote:

> In views/guide.lavender
> <https://github.com/golavender/golavender.github.io/pull/57#discussion_r26052691>
> :
>
> > @@ -89,8 +89,9 @@ block content
> >            h2#github via Github
> >            p
> >              | Add the lavender submodule with
> > -            code git submodule add git@github.com:golavender/lavender.git (folder to
> > -            | clone to) and include it with
> > +            code git submodule add git@github.com:golavender/lavender.git (folder to
> > +            code clone to)
>
> Cool tidbit here for you @jennEDVT <https://github.com/jennEDVT> , if
> it's a small change like this, you can just do a quick edit in GitHub. Go
> to the 'Files Changed' tab, and click on the pencil to the left of the file
> name (it should pop up saying something about file changes).
> Do a quick fix, commit the changes, and it's done. Just don't go crazy and
> do lines and lines of code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/golavender/golavender.github.io/pull/57/files#r26052691>
> .
>